### PR TITLE
GitPod config with Dockerfile

### PR DESCRIPTION
**Pull request by axmmisaka**
Current configuration with .gitpod.yml requires the user to wait a while after gitpod workspace indicates "ready" which is not good according to @lhstrh. This change allows gitpod to not show stuff before time-consuming configurations are completed. This is also a crucial step towards building a docker image that supports LF development.

To test: https://gitpod.io/new/#https://github.com/axmmisaka/lingua-franca-playground/tree/gitpod-docker

or https://gitpod.io/new/#imagebuild/https://github.com/axmmisaka/lingua-franca-playground/tree/gitpod-docker
to trigger force rebuild

(Building image takes looooooooooooooong, let's see if we can prebuild image)

### CODE REVIEW

**Comment by lhstrh** [comment]
We could publish an image on the Docker registry? We already have an active org there...

**Comment by lhstrh** [comment]
When I opened the first link, it must have used a pre-built image because it came up really fast!

**Comment by axmmisaka** [comment]
> We could publish an image on the Docker registry? We already have an active org there...

We could...... But I guess the only benefit would be that GitHub would be able to use it as well.



> When I opened the first link, it must have used a pre-built image because it came up really fast!

Yes! I think that's what GitPod recommends us to do as well.

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+image:
+  file: .gitpod-base.Dockerfile
+
+## Example is handled separately in GitPod for faster initialisation.
```
Comment (lhstrh) [inline .gitpod.yml:4]:
```suggestion
## Example is handled separately in GitPod for faster initialization.
```

**Review by lhstrh — APPROVED** [review]
This looks great, @axmmisaka! Thank you so much for your help with this!

**Comment by lhstrh** [comment]
I guess one thing that's left to do is update the `README.md` with instructions for firing up a GitPod using this configuration.

**Pull request merged**